client/js/models/WhereToFindUs.js

```diff
@@ -0,0 +1,19 @@
+module.exports = { ...require('./__proto__.js'),
+	data: {
+		marketData: [
```

Comment:
We have an empty database right now, so don't worry about it for now, but once we have one, we will fetch the data for the farmers markets, and for each, create the HTML.

Reply:
This is great, just use fake data on the front end, and when everything looks good we will plug in the backend.